plugins/modules/kms_key_info.py

```diff
@@ -143,7 +143,7 @@
       returned: always
       sample: "My Key for Protecting important stuff"
     enabled:
-      description: Whether the key is enabled. True if I(key_state) is C(Enabled).
+      description: Whether the key is enabled. True if RV(kms_keys.key_state) is C(Enabled).
```

Comment:
```suggestion
      description: Whether the key is enabled. True if O(kms_keys.key_state) is V(Enabled).
```

Reply:
key_state is only a returned parameter, not a module's option (that's why O() wouldn't work).